toadr3/report.py

```diff
@@ -125,3 +127,113 @@ def payload_descriptors(self) -> list[ReportPayloadDescriptor] | None:
     def resources(self) -> list[ReportData]:
         """The resources of the report."""
         return self._resources
+
+
+def create_report(
```

Comment:
If I understand correctly, we send a reports to the VTN to update them about the power usage. Could you add some documentation about the use of the report and the fields? I think report_values contains the measured power values. But other than that I have some issues understanding what this is used for and how. This is 100% my lack of knowledge about ADR, but that makes it difficult to use the library :)